shub/settings.py

```diff
@@ -0,0 +1,537 @@
+    "VIEW_RATE_LIMIT_BLOCK": True,
+}
+
+STRING_DEFAULTS = {
+    "API_VERSION": "v1",
+    "API_ANON_THROTTLE_RATE": "100/day",
+    "API_USER_THROTTLE_RATE": "1000/day",
+    "API_DEFAULT_SCHEMA_CLASS": "rest_framework.schemas.coreapi.AutoSchema",
+    "API_DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
+    "SOCIAL_AUTH_LOGIN_REDIRECT_URL": "http://127.0.0.1",
+    # On any admin or plugin login redirect to standard social-auth entry point for agreement to terms
+    "LOGIN_REDIRECT_URL": "/login",
+    # DOMAIN NAMES
+    ## IMPORTANT: if/when you switch to https, you need to change "DOMAIN_NAME"
+    # to have https, otherwise some functionality will not work (e.g., GitHub webhooks)
+    "DOMAIN_NAME": "http://127.0.0.1",
+    "DOMAIN_NAME_HTTP": "http://127.0.0.1",
+    # Get additional admins from the environment
+    "HELP_CONTACT_EMAIL": "vsoch@users.noreply.github.com",
+    "HELP_INSTITUTION_SITE": "https://srcc.stanford.edu",
+    "REGISTRY_NAME": "Tacosaurus Computing Center",
+    "REGISTRY_URI": "taco",
+    # "UA-XXXXXXXXX"
+    "GOOGLE_ANALYTICS": None,
+    "GOOGLE_BUILD_SINGULARITY_VERSION": "v3.3.0-slim",
+    "SREGISTRY_DATABASE_ENGINE": "django.db.backends.postgresql_psycopg2",
+    "SREGISTRY_DATABASE_NAME": "postgres",
+    "SREGISTRY_DATABASE_USER": "postgres",
+    "SREGISTRY_DATABASE_HOST": "db",
+    "SREGISTRY_DATABASE_PORT": "5432",
+    # Minio Storage
+    "MINIO_SERVER": "minio:9000",  # Internal to sregistry
+    "MINIO_EXTERNAL_SERVER": "127.0.0.1:9000",  # minio server for Singularity to interact with
+    "MINIO_BUCKET": "sregistry",
+    "MINIO_REGION": "us-east-1",
+    # The rate limit for each view, django-ratelimit, "50 per day per ipaddress)
+    "VIEW_RATE_LIMIT": "50/1d",
+    "DJANGO_LOG_LEVEL": "WARNING",
+    # An image url or one of the following: 'mm', 'identicon', 'monsterid', 'wavatar', 'retro'. Defaults to 'mm'
+    "GRAVATAR_DEFAULT_IMAGE": "retro",
+}
+
+INTEGER_DEFAULTS = {
+    "API_PAGE_SIZE": 10,
+    # Set this to be some size in MB to limit uploads.
+    # Uploads > 2.5GB will not use memory, but the filesystem
+    "DATA_UPLOAD_MAX_MEMORY_SIZE": None,
+    # Limit users to N collections (None is unlimited)
+    "USER_COLLECTION_LIMIT": 2,
+    # The number of collections to show on the /<domain>/collections page
+    "COLLECTIONS_VIEW_PAGE_COUNT": 250,
+    # The maximum number of downloads allowed per container/collection, per week
+    "CONTAINER_WEEKLY_GET_LIMIT": 100,
+    "COLLECTION_WEEKLY_GET_LIMIT": 100,
+    "GOOGLE_BUILD_LIMIT": 100,
+    "GOOGLE_BUILD_TIMEOUT_SECONDS": None,  # None defaults to 10 minutes
+    "GOOGLE_BUILD_EXPIRE_SECONDS": 28800,
+    "CONTAINER_SIGNED_URL_EXPIRE_SECONDS": 10,
+    "MINIO_SIGNED_URL_EXPIRE_MINUTES": 5,
+}
+
+# Environment helpers
+
+
+def get_sregistry_envar(key):
+    envar_key = "SREGISTRY_%s" % key
+    return os.environ.get(envar_key)
+
+
+def get_sregistry_envar_list(key):
```

Comment:
The only 2 places where this is used already prefix `key` with `SREGISTRY_`. Consider doing the prefixing within this function like you do for `get_sregistry_envar()` so they are parallel and the caller doesn't need to prefix it.